src/content/community.md

```diff
@@ -52,6 +53,39 @@ There are several ways to stay up-to-date with Firefox add-on technology and con
 
 Stay up to date with the latest information about add-on technology on the [Add-ons blog](https://blog.mozilla.org/addons).
 
+### Office hours <span id="office-hours"></span>
+
+The Firefox Add-ons team holds one-on-one office hours sessions to help developers make their extensions available on Android. Developers can also use the [Firefox for Android Add-ons](https://discourse.mozilla.org/c/add-ons/android/9393) forum to discuss issues asynchronously.
+
+<!-- Google Calendar Appointment Scheduling begin -->
+<link href="https://calendar.google.com/calendar/scheduling-button-script.css" rel="stylesheet">
+<script src="https://calendar.google.com/calendar/scheduling-button-script.js" async></script>
+<script>
+(function() {
+  var target = document.currentScript;
+  window.addEventListener('load', function() {
+    calendar.schedulingButton.load({
+      url: 'https://calendar.google.com/calendar/appointments/schedules/AcZssZ3yNnoZNBdJCTiJjqziXs9GoCYpfW9XZFkB0QWKNy9dfoqqPlCw7UsYh48Ff0i3lZcA5YzygwrU?gv=true',
+      color: '#039BE5',
+      label: 'Book an appointment',
```

Comment:
To align with earlier wording
```suggestion
      label: 'Book a session',
```

Reply:
This widget has been dropped in response to @Rob--W's feedback. The language used to open the new UI element is "Book a session".